src/allotropy/parsers/unchained_labs_lunatic/unchained_labs_lunatic_reader.py

```diff
@@ -0,0 +1,11 @@
+from io import IOBase
+
+import numpy as np
+import pandas as pd
+
+
+class UnchainedLabsLunaticReader:
+    def __init__(self, contents: IOBase):
+        self.data = pd.read_csv(  # type: ignore[call-overload]
```

Comment:
We really shouldn't need to do these suppressions. Clearly IOBase is an invalid type. I'll try to sort it out separately.